package.json

```diff
@@ -23,15 +23,15 @@
   "author": "Palantir",
   "license": "Apache-2.0",
   "dependencies": {
-    "stylelint-config-standard": "^16.0.0",
+    "stylelint-config-standard": "^17.0.0",
     "stylelint-order": "^0.3.0"
   },
   "devDependencies": {
-    "stylelint": "7.9.0",
-    "stylelint-scss": "1.4.3"
+    "stylelint": "8.0.0",
+    "stylelint-scss": "1.5.1"
   },
   "peerDependencies": {
-    "stylelint": "^7.9.0"
+    "stylelint": "^7.12.0 || ~8.0.0"
```

Comment:
That [semantic versioning policy](https://github.com/stylelint/stylelint/blob/8.0.0/README.md#semantic-versioning-policy) is for applications, not config libraries like this. Basically what you've done here is prevent our apps from upgrading to 8.1, which is not what we want. The new policy isn't entirely novel; like they said it's like other code quality tools, including TSLint. Based on my experience with TSLint I think you should make this `^7.12.0 || `^8.0.0`.